Makefile

```diff
@@ -866,3 +866,49 @@ install-git-hooks:
 .PHONY: pre-commit
 pre-commit:
 	$(CONTAINERIZED) $(CALICO_BUILD) git-hooks/pre-commit-in-container
+
+CRS_VERSION := 3.3.5
+MODSEC_VERSION := v3.0.10
+
+CRS_EMBED_FOLDER := pkg/render/applicationlayer/embed/coreruleset
+CRS_ARCHIVE_SRC := https://github.com/coreruleset/coreruleset/archive/refs/tags/v$(CRS_VERSION).tar.gz
+CRS_ARCHIVE_SHA := b552005471446a4a2454a6b6ee7e65f864219ce0
+CRS_ARCHIVE_TAR := v$(CRS_VERSION).tar.gz
+CRS_ARCHIVE_TMP := $(shell mktemp -d)
```

Comment:
I don't like that this will create a tmp directory on every make invocation. It would be much better if you had this done in the `$(CRS_EMBED_FOLDER)` target since it is only used in there and could be cleaned it up in there too.